src/Illuminate/Container/Container.php

```diff
@@ -1238,4 +1238,21 @@ public function __set($key, $value)
     {
         $this[$key] = $value;
     }
+
+    /**
+     * Get a closure to resolve the given type from the container.
+     * The given closure will have an optional [array $parameters] parameter
+     * to merge with the default parameters given when the factory was made.
+     *
+     * @param  string $abstract
```

Comment:
Please add the missing space before the dollar. There should be exactly 2 space.